diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -298,6 +314,27 @@ namespace diff_drive_controller
 
   void DiffDriveController::update(const ros::Time& time, const ros::Duration& period)
   {
+    // UPDATE DYNAMIC PARAMS
+    // Retreive dynamic params:
+    DynamicParams dynamic_params = *(dynamic_params_.readFromRT());
```

Comment:
This is going to update all these parameters on control cycle, which seems like a lot. Would it be safe to use a boolean flag to notify of update?